TAF_EasyRest/TestFramework/Test/ClientTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using TestFramework.Tools.GetData;
+
+namespace TestFramework.Test
+{
+    [TestFixture]
+    public class ClientTests : BaseTest
+    {
+        public IWebDriver driver { get; private set; }
+        private string email;
+        private string password;
+
+        [OneTimeSetUp]
+        public void BeforeAllTests()
+        {
+            driver = new ChromeDriver();
+            SignInPage signInPage = new SignInPage(driver);
```

Comment:
Do you need this page here? Later you have the UserLogin method with that page inside.